package.json

```diff
@@ -20,7 +20,7 @@
     "test": "test"
   },
   "scripts": {
-    "tsc": "rm -rf build/*/** && tsc",
+    "tsc": "rm -rf build && tsc",
```

Comment:
How?  Tsc generate those subfolders but not top level build so its fair to clean only what is regenerated.

Your proposed change will break linked editor and voyager every time you run test.